tools/operator/README.md

```diff
@@ -0,0 +1,107 @@
+# Fabric Network Operator
+
+- A tool to launch fabric network on kubernetes cluster or local machine with a docker-compose
+file using a network input file and gives back connection profiles for each peer organization
+to use with any client. This tool also uses a testclient to perform the fabric operations which
+includes creating a channel, joining peers to a channel, anchor peer update, installing, upgrading
+and instantiating the chaincode, performing invokes and queries, based on a test input file. This tool is also used for creating channelTx, migrating the network to etcdraft, checking the health of peers and orderers and upgrading the network on providing a specific network input file. This uses `ytt` 
+<https://github.com/k14s/ytt/blob/master/README.md> to generate all necessary configuration files 
+and a go program to launch fabric network
+
+## Prerequisites
+
+- Go 1.11.0 or above <https://golang.org/dl/>
+- yaml.v2 go package (go get gopkg.in/yaml.v2)
+- Fabric binaries in $PATH <https://hyperledger-fabric.readthedocs.io/en/latest/install.html>
+
+#### Kubernetes
+
+- Kubernetes cluster if launching fabric network on kubernetes cluster. It is the responsibility
+of the user to carefully consider and to create a kube cluster with enough resources to handle
+the number of nodes specified in networkspec for the planned traffic patterns and rates
+- `kubectl` if launching fabric network on kubernetes cluster. Refer to
+<https://kubernetes.io/docs/tasks/tools/install-kubectl/> for installing `kubectl`
+
+#### Local using docker
+
+- Docker <https://docs.docker.com/install/linux/docker-ce/ubuntu/> and Docker-compose
+<https://docs.docker.com/compose/install/> if launching fabric network locally
+
+## Usage
+
+Supported input arguments for launcher are:
+
+```go
+-a string
+       Set action(up, down, create, join, anchorpeer, install, instantiate, upgrade, 
+	   invoke, query, createChannelTxn, migrate, health) (default is up)
+-i string
+       Network spec input file path (Required)
+-k string
+       Kube config file path (If omitted, then use local network)
```

Comment:
kubeconfig is a single word. Apply this wherever possible.  Mention that, default to local if we don't specify -k argument.

Reply:
Kube config means the Kubernetes Config file. kubeconfig is just the name of an env variable